presto-spark-classloader-interface/src/main/java/com/facebook/presto/spark/classloader_interface/PrestoSparkTaskRdd.java

```diff
@@ -56,20 +56,20 @@
  * <p>
  * The broadcast inputs are encapsulated in taskProcessor.
  */
-public class PrestoSparkTaskRdd
-        extends ZippedPartitionsBaseRDD<Tuple2<MutablePartitionId, PrestoSparkMutableRow>>
+public class PrestoSparkTaskRdd<T>
```

Comment:
Add a comment that T should only be PrestoSparkMutableRow or PrestoSparkSerializedPage when in use.

Reply:
Added a marker interface, `PrestoSparkTaskOutput`. Now it should be enforced by the compiler.